hackathon_site/dashboard/frontend/src/api/types.ts

```diff
@@ -88,6 +88,7 @@ export interface Team {
 }
 
 /** Orders API */
+export type OrdersOrdering = "" | "timeOrderedASC" | "timeOrderedDESC" | "OrderQuality";
```

Comment:
I think you're only using this in OrderFilter.tsx so you can move it there and leave this file unchanged